### PR TITLE
Analytics Revamp

### PR DESCRIPTION
**Pull request by smitch88**
**Analytics Site port into ousd.com**

**High-level Changes**
- Refactor .js -> .ts across the board
- Added `wagmi` config
  - Gas tracker on mainnet
- Added `getLayout` for page oriented layout changes (e.g, main vs analytics)
- Added new `/analytics` route
- Added Dune api integration client
- Added sub-pages for the following:
  - Overview
    - APY / Total Supply / OUSD relative to ETH / Collateral charts
  - Collateral
    - Stablecoin distribution
    - Chart
  - Protocol Revenue
    - Daily, weekly, total rev
    - Chart
  - Strategies
    - Yield source distribution 
  - Health Monitoring
    - Iframes from grafana
- Added mobile menu layout

**Whats not done and will be done in follow-up**
- Caching layer for Dune queries
- Supply breakdown page
- Graph filtering by query parameter
- Add OGV and OUSD prices in `RealTimeStats` component

**Screenshots**
Screenshot 2023-03-30 170546
Screenshot 2023-03-30 170617
Screenshot 2023-03-30 170636
Screenshot 2023-03-30 170712
Screenshot 2023-03-30 170728

### CODE REVIEW

**Comment by HrikB** [comment]
I'm having some issues on the top level directory. `/analytics` loads fine on the initial render but going back to the root directory (`/analytics` -> `/analytics/collateral` `/analytics`) crashes the app for me.

**Comment by HrikB** [comment]
@micahalcorn We may want to add some sort of note about ad blocker. The page was loading a blank for me initially because of my ad blocker.

File: pages/analytics/strategies.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+import { strategyMapping } from "../../src/utils/constants";
+import { formatCurrency } from "../../src/utils/math";
+
+const protocolToDetails = {
+  Morpho: {
+    description: `Morpho adds a peer-to-peer layer on top of Compound and Aave allowing
+          lenders and borrowers to be matched more efficiently with better
+          interest rates. When no matching opportunity exists, funds flow
+          directly through to the underlying protocol. OUSD supplies stablecoins
+          to three of Morpho’s Compound markets to earn interest. Additional
```
Comment (HrikB) [inline pages/analytics/strategies.tsx:22]: For apostrophes, use `&apos;`. Otherwise build will fail in heroku.

**Review by HrikB — APPROVED** [review]
Overall looks great. The only thing I'd wanna make sure is that routing back to the top-level directory is still possible. But other than that I think this is fine to merge.

File: pages/analytics/strategies.tsx (hunk 1 above)
Comment (smitch88) [reply pages/analytics/strategies.tsx:22]: This will even surrounded by back ticks?

**Pull request merged**